ktor-client/ktor-client-core/common/src/io/ktor/client/utils/exceptions.kt

```diff
@@ -0,0 +1,14 @@
+/*
```

Comment:
filename

Reply:
I'm not sure I understand your point, could you clarify it?